collector/lib/CollectorStats.h

```diff
@@ -14,22 +14,29 @@
   X(net_create_message) \
   X(net_write_message)
 
-#define COUNTER_NAMES             \
-  X(net_conn_updates)             \
-  X(net_conn_deltas)              \
-  X(net_conn_inactive)            \
-  X(net_cep_updates)              \
-  X(net_cep_deltas)               \
-  X(net_cep_inactive)             \
-  X(net_known_ip_networks)        \
-  X(net_known_public_ips)         \
-  X(process_lineage_counts)       \
-  X(process_lineage_total)        \
-  X(process_lineage_sqr_total)    \
-  X(process_lineage_string_total) \
-  X(process_info_hit)             \
-  X(process_info_miss)            \
-  X(rate_limit_flushing_counts)
+#define COUNTER_NAMES                       \
+  X(net_conn_updates)                       \
+  X(net_conn_deltas)                        \
+  X(net_conn_inactive)                      \
+  X(net_cep_updates)                        \
+  X(net_cep_deltas)                         \
+  X(net_cep_inactive)                       \
+  X(net_known_ip_networks)                  \
+  X(net_known_public_ips)                   \
+  X(process_lineage_counts)                 \
+  X(process_lineage_total)                  \
+  X(process_lineage_sqr_total)              \
+  X(process_lineage_string_total)           \
+  X(process_info_hit)                       \
+  X(process_info_miss)                      \
+  X(rate_limit_flushing_counts)             \
+  X(procfs_could_not_open_fd_dir)           \
```

Comment:
There is a recently merged documentation containing list of metrics, could you pls update it as well?

Reply:
Done